src/main/resources/wa-task-configuration-wa-wacasetype.dmn

```diff
@@ -1794,6 +1794,41 @@ else if (taskAttributes != null and taskAttributes.additionalProperties != null
           <text></text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_0sd5q7s">
+        <inputEntry id="UnaryTests_1ma9i3h">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_14040yd">
+          <text>"reconfigTaskAttributesTask"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0ycu0e0">
+          <text>"additionalProperties_roleAssignmentId"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1clkfme">
+          <text>if (taskAttributes != null and taskAttributes.roleAssignmentId != null) then taskAttributes.roleAssignmentId
+else if (taskAttributes != null and taskAttributes.additionalProperties != null and taskAttributes.additionalProperties.roleAssignmentId != null) then taskAttributes.additionalProperties.roleAssignmentId else ""</text>
```

Comment:
Is this from another change?  Seems to be about the roleAssignmentId rather than process category